src/scheduler/pbsfs.c

```diff
@@ -88,7 +88,8 @@ static void print_fairshare(group_info *root, int level);
 int
 main(int argc, char *argv[])
 {
-	char path_buf[256];
+	char path_buf[256] = {0};
+	char sched_name[PBS_MAXSCHEDNAME + 1] = {0};
```

Comment:
Maybe you want to initialize this to "default" so it will contain a valid scheduler name when -I isn't specified?